src/Bedrock.Framework.Experimental/Protocols/Http1RequestMessageReader.cs

```diff
@@ -54,7 +54,7 @@ public bool TryParseMessage(in ReadOnlySequence<byte> input, ref SequencePositio
                     goto case State.Headers;
 
                 case State.Headers:
-                    while (sequenceReader.TryReadTo(out var headerLine, NewLine))
+                    while (sequenceReader.TryReadTo(out ReadOnlySequence<byte> headerLine, NewLine))
```

Comment:
Undo

Reply:
On .NET 6 you get the following error when you use the `var`:
```
Error	CS0121	The call is ambiguous between the following methods or properties: 
'SequenceReader<T>.TryReadTo(out ReadOnlySequence<T>, ReadOnlySpan<T>, bool)' and
'SequenceReader<T>.TryReadTo(out ReadOnlySpan<T>, ReadOnlySpan<T>, bool)'
```